### PR TITLE
My fixes

### PR DESCRIPTION
**Pull request by ghost**
I
- switched to Shoutrrr to send form notifications as it's more clean and has support for more services.
- Made the dockerfile smaller.
- Removed the webhook test thing.
- Did not test any of my changes. (yet)
- Fixed formatting on some files

### CODE REVIEW

**Comment by Midou36O** [comment]
Isn't shoutrr made to send webhooks and notifications?

**Comment by ghost** [comment]
> Isn't shoutrr made to send webhooks and notifications?

Yes! That's why I switched to it. It also looks more clean on the code side, and allows us to send to multiple services in the future if you'd want that.

File: data/config.example.toml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 [segfautils]
 port = 6893
-webhook_url = "YOURWEBHOOKURL"
```
Comment (Midou36O) [inline data/config.example.toml:3]: Wait wait why remove the webhook config? Where will it go now?

**Review by Midou36O — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
we're switching to shoutrrr, that's whySendt fra min Galaxy
-------- Opprinnelig melding --------Fra: Midou36O ***@***.***> Dato: 18.10.2022  17:55  (GMT+01:00) Til: ProjectSegfault/Segfautils ***@***.***> Ko: Odyssey346 ***@***.***>, Author ***@***.***> Emne: Re: [ProjectSegfault/Segfautils] My fixes (PR #15) 
@Midou36O requested changes on this pull request.



In data/config.example.toml:
> @@ -1,6 +1,5 @@
 [segfautils]
 port = 6893
-webhook_url = "YOURWEBHOOKURL"

Wait wait why remove the webhook config? Where will it go now?

**Review by Midou36O — APPROVED** [review]

File: data/config.example.toml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 [segfautils]
 port = 6893
 auth_token = "YOURAUTHTOKEN"
+shoutrrr_url = "YOURSHOUTRRURL"
```
Comment (ghost) [inline data/config.example.toml]: No

**Comment by ghost** [comment]
image
Btw midou if you were wondering how ``shoutrrr`` looks on Discord. It'll look different on Matrix

**Comment by Midou36O** [comment]
Shoutrr supports matrix so it's fine

File: data/config.example.toml (hunk 2 above)
Comment (Midou36O) [reply data/config.example.toml]: Why not?

Comment (Midou36O) [reply data/config.example.toml]: I don't see where the link is pointing to

Comment (ghost) [reply data/config.example.toml]: I would rather have a [Service URL](https://containrrr.dev/shoutrrr/v0.5/services/overview/) as the default example.

Comment (Midou36O) [reply data/config.example.toml]: hmm, and how do you think you can implement that?

Comment (ghost) [reply data/config.example.toml]: ``shoutrrr_url = "discord://token@id" # Shoutrrr URL examples are available at https://containrrr.dev/shoutrrr/v0.5/services/overview``

**Comment by ghost** [comment]
bump?

**Comment by Midou36O** [comment]
I'm waiting for the PR to be ready for review to actually review it and Merge it

**Comment by ghost** [comment]
ah ok

**Comment by ghost** [comment]
Can you push the change I asked you to do, though?

**Comment by ghost** [comment]
Then I will mark it for review.

**Comment by Midou36O** [comment]
> Can you push the change I asked you to do, though?

Which ones already?

**Comment by Midou36O** [comment]
Actually, i'll add a coment next to the shoutrr like you indicated so that the user knows what it can do

**Review by Midou36O — APPROVED** [review]
LGTM :shipit:

**Comment by ghost** [comment]
what the hell is that goofy ahh emoji

**Comment by Midou36O** [comment]
GitHub one, it's been there for a long time lol

**Pull request merged**